### ISSUE TITLE
9.1 undefined offset when login failed

### ISSUE THREAD

**Issue by mmattel** [owncloud/core#26013]
### Steps to reproduce
1. logon with a unknown user or wrong password

### Expected behaviour
No errors beeing logged

### Actual behaviour
Errors beeing logged

### Server configuration
**Operating system**: Ubuntu 14.04

**Web server:** nginx

**Database:** mysql

**PHP version:** 5.5.9

**ownCloud version:** 9.1.0 (stable)

**Updated from an older ownCloud or fresh install:** fresh

**Where did you install ownCloud from:** tar

**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.3.2
  - comments: 0.3.0
  - dav: 0.2.5
  - federatedfilesharing: 0.3.0
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.6.0
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.10.0
  - files_texteditor: 2.1
  - files_trashbin: 0.9.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 15.0.0
  - logreader: 1.1.5
  - notifications: 0.3.0
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.1
  - updatenotification: 0.2.1
Disabled:
  - encryption
  - external
  - files_antivirus
  - user_external
  - user_ldap
```

**The content of config/config.php:**

```
{
    "system": {
        "instanceid": "oct7bkicorxf",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "xxxx"
        ],
        "datadirectory": "\/var\/www\/owncloud\/data",
        "overwrite.cli.url": "https:\/\/xxxx",
        "dbtype": "mysql",
        "version": "9.1.0.15",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "Europe\/Vienna",
        "installed": true,
        "filelocking.enabled": "true",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "localhost",
            "port": 6379,
            "timeout": 0,
            "dbindex": 0
        },
        "filesystem_check_changes": 1,
        "loglevel": 1,
        "mail_smtpmode": "smtp",
        "mail_from_address": "xxx",
        "mail_smtpsecure": "tls",
        "mail_domain": "xxx",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtphost": "xxx",
        "mail_smtpport": "587",
        "maintenance": false
    }
}
```

**Are you using external storage, if yes which one:** local/smb/sftp/... yes

**Are you using encryption:**no  

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/... no

### Client configuration
**Browser:** Opera

**Operating system:** W7x64

#### ownCloud log (data/owncloud.log)
```
Error	PHP	Invalid argument supplied for foreach() at /var/www/owncloud/core/templates/login.php#23
Error	PHP	Undefined offset: 1 at /var/www/owncloud/core/Controller/LoginController.php#117
Warning	core	Login failed: lulu' (Remote IP: 'xx.xx.xx.xx')
```

**Comment by RealRancor** [comment]
@mmattel Duplicate of https://github.com/owncloud/core/issues/25714 and already fixed in 9.1.1

**Comment by mmattel** [comment]
Thanks, closing
